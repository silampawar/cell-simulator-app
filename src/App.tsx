
import CellSimulator from './Components/CellSimulator';
import {AppContainer,AppHeader} from "./App.styles"

function App() {
  return (
    <div>
      <AppHeader>
        <div>Cell Simulator Game</div>
      </AppHeader>
      <main>
        <AppContainer>
          <CellSimulator/>
        </AppContainer>
      </main>
    </div>
  );
}

export default App;
