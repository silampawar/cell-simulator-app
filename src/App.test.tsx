import App from './App';
import { shallow } from "enzyme";

test('renders App component link', () => {

  const wrapper = shallow(<App/>);
  expect(wrapper).toMatchSnapshot();

});
