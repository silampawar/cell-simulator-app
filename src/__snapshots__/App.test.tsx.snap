// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders App component link 1`] = `ShallowWrapper {}`;
